active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,655 @@
+* void overridable OnWindowCreated (WindowCreatedEventArgs)
+* EnteredBackgroundEventHandler EnteredBackground;
+* LeavingBackgroundEventHandler LeavingBackground;
+* SuspendingEventHandler Suspending;
+* EventHandler&lt;Object&gt; Resuming
+
+
+## Application.Start method
+_Behavior update to existing API when running as Desktop app_
+
+Static method which provides the entry point and requests initialization of the application. Use the callback 
+to instantiate the Application class.
+
+In a Desktop app this will run a message pump internally, and not return until the application 
+shuts down. In a UWP app it will return immediately.
+
+
+## Application.OnLaunched(XamlLaunchActivatedEventArgs) method
+_Behavior update to existing API when running as Desktop app_
+
+Invoked when the application is launched. Override this method to perform application initialization 
+and to display initial content in the associated Window (UWP apps) or create a new Window (Desktop apps).
+
+```CS
+protected virtual void OnLaunched(XamlLaunchActivatedEventArgs args)
+```
+
+> This method is typically created automatically when a project is created. 
+You can then modify it or remove it.
+
+### Examples
+
+In a Desktop app, use OnLaunched to create the main window and pass it the first command line argument.
+
+```CS
+protected override void OnLaunched(XamlLaunchActivatedEventArgs e)
+{
+     MainWindow mainWindow = new MainWindow();
+     
+     if(e.Arguments != null)
+     {
+        string[] arguments = e.Arguments.Split(' ');
+        // Assuming MainWindow has a public property called FileName 
+        mainWindow.FileName = arguments[0];
+     }	
+
+     mainWindow.Activate();
+}
+```
+
+In a UWP app,
+
+```CS
+protected override void OnLaunched(XamlLaunchActivatedEventArgs e)
+{
+     MainPage page = new MainPage();
+     
+     if(e.Arguments != null)
+     {
+        string[] arguments = e.Arguments.Split(' ');
+	    // Assuming MainPage has a public property called FileName 
+        page.FileName = arguments[0];
+     }	
+
+    Window.Current.Content = page;
+    Window.Current.Activate();
+    window.Activate();
+}
+```
+
+
+## Application.RequiresPointerMode property
+_Behavior update to existing API when running as Desktop app_
+
+Gets or sets whether a UWP app supports mouse mode, which emulates pointer interaction experiences 
+with non-pointer input devices such as an Xbox gamepad or remote control. (All nested elements inherit 
+this behavior.)
+
+> Note: this property is ignored in a Desktop app
+
+## Application.OnWindowCreated(XamlWindowCreatedEventArgs) method
+_Behavior update to existing API when running as Desktop app_
+
+Invoked when the application creates a window.
+
+```CS
+protected virtual void OnWindowCreated(XamlWindowCreatedEventArgs args)
+```
+
+### Remarks
+- In a UWP app, this method is called once for the creation of the main window, once for each hosted window, 
```

Comment:
For a UWP app, does this mean that OnWindowCreated is called before OnLaunched? This would be a gotcha case for many apps, since they typically do the bulk of their initialization in OnLaunched. May want to add a Note for this case.